README.md

```diff
@@ -50,8 +50,6 @@ using the special address `host.docker.internal`. This is not recognised by Dock
 can be overriden by the environment variable `DOCKER_GATEWAY_HOST`:
 
 ```bash
-$ DOCKER_GATEWAY_HOST="`hostname -I` |awk '{print $1}'`" npm start
-```
-
-**TODO**: Another OSX/Linux incompatibility is related to the aws development configuration, where OSX expects
-an url while Linux expects a host name.
\ No newline at end of file
+EXPORT DOCKER_GATEWAY_HOST=`docker network inspect --format='{{range .IPAM.Config}}{{.Gateway}}{{end}}'
```

Comment:
Could you explain me a bit of what this does? Does this work or additional steps should be taken to solve the other linux incompatibility?

Reply:
I think it should be enough 🤞 

```
$ docker network inspect --format='{{range .IPAM.Config}}{{.Gateway}}{{end}}
```
should return the ip that works on linux. I've tried it in OSX and it worked, and I have seen reports of it working in linux. With the backticks, we execute the command and assign its output to `DOCKER_GATEWAY_HOST` env var. Would be good to try the command above in your environments before approving...

Once the `DOCKER_GATEWAY_HOST` environment variable is set to the proper value, the local starter should launch the client docker container with it also set to the same value. There, that value is used by the R code whenever the payload has a request for `"server": "host.docker.internal"`

Reply:
https://stackoverflow.com/a/57127242/239408